packages/@react-spectrum/tag/src/Tag.tsx

```diff
@@ -36,7 +36,6 @@ export function Tag<T>(props: SpectrumTagProps<T>) {
   let {styleProps} = useStyleProps(otherProps);
   let {hoverProps, isHovered} = useHover({isDisabled});
   let {isFocused, isFocusVisible, focusProps} = useFocusRing({within: true});
-  let labelRef = useRef();
   let tagRef = useRef();
   let tagRowRef = useRef();
```

Comment:
just to make sure i'm understanding you, you don't like that ref is being passed through using useTag and would rather it just apply it to the divs directly because they aren't being used in any special way in the hook?

Reply:
Right, I think it isn't necessary to return the `tagRef` via the props that are returned by `useTag` since the end user should be attaching that ref to their tag element themselves. 

Furthermore, it is important to attach the `tagRef` and `tagRowRef` to their respective elements in `Tag.tsx` since `useTag` provides those refs to `useGridCell` and `useGridRow` which in turn pass those refs to `useSelectableItem` so that the proper dom node can get focused when `state.selectedKey` matches the row/cell key. Stuff might break due to that but I'm hopeful that the `focusMode` will ensure that row focus never happens